AirCasting/UIState/UIStateHandler.swift

```diff
@@ -0,0 +1,35 @@
+// Created by Lunar on 10/02/2022.
+//
+
+import Foundation
+import Resolver
+
+protocol UIStateHandler {
```

Comment:
`SessionCardUIStateHandler` maybe? By naming it this way we discourage from putting more and more methods here and instead create separate interfaces for separate parts